molecule/default/converge.yml

```diff
@@ -11,6 +11,20 @@
       ansible.builtin.command: "mysql -u root -proot -e 'show databases;'"
       changed_when: false
 
-    - name: Make sure we can connect to MySQL via TCP.
+    - name: Make sure we can connect to MySQL via TCP on old MySQL version.
       ansible.builtin.command: "mysql -u root -proot -h 127.0.0.1 -e 'show databases;'"
```

Comment:
Can we still have a command to try connecting via socket separately to 'show databases' so we have this test for all versions of MySQL/MariaDB?